run/run.py

```diff
@@ -113,10 +119,18 @@ def main(platform_id, platform, args, config):
 
     patch_wpt(config, platform)
 
-    # TODO(#40): modify this to test against the first SHA of the day
-    CURRENT_WPT_SHA = get_current_wpt_sha(config)
+    wpt_path = config['wpt_path']
+    sha_finder = shas.SHAFinder(logger)
+    CURRENT_WPT_SHA = (sha_finder.get_todays_sha(wpt_path)
```

Comment:
Rename this to something more accurate, like just WPT_SHA, or maybe CHOSEN_WPT_SHA?

Reply:
Done.